webapp/src/errorList.tsx

```diff
@@ -29,32 +29,40 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
     }
 
     render() {
-        const showCollapseButton = true;
+        const {isCollapsed, errors} = this.state;
+        const errorsAvailable = errors.length != 0;
```

Comment:
Is error guaranteed to be defined? Assume it’s not (being defensive is good).

    const errorsAvailable = !!errors?.length

Reply:
Ok just tweaked that